src/schemas/committee.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from "zod";
+
+export const schema = z.object({
+  name: z.string().min(1, { message: "Name is required" }),
+  email: z.string().email({ message: "Invalid email address" }),
+  discord: z.string().min(1, { message: "Discord username is required" }),
+  major: z.string().min(1, { message: "Please select your major" }),
+  grade: z.string().min(1, { message: "Please select your grade" }),
+  gender: z.enum(["Male", "Female", "Transgender", "Non-binary", "Other"], {
+    required_error: "Please select your gender",
+  }),
+  shirt: z.enum(["XS", "S", "M", "L", "XL", "XXL"], {
```

Comment:
u can pull these from the existing files, these enums already exist in information.js